app/models/spree/digital_link.rb

```diff
@@ -34,12 +33,20 @@ def reset!
       update_column :created_at, Time.now
     end
 
+    def attachment
+      if digital.drm?
+        digital.drm_records.find_by(line_item: line_item)&.attachment
```

Comment:
It is using in template after user already payed for order, we can't show 500 there